### DISCUSSION TITLE
"Xcode Command Line Tools" is bundled with brew to install ?

### DISCUSSION THREAD

**Discussion by dayDPTR** [Homebrew/discussions#3426] [Getting started]
hi Team, one prompt(see below log) confuse me while installing brew tool at MacOS.

```
MARIO-M-3MN6:~ mario$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Password:
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew
==> The following new directories will be created:
/opt/homebrew/bin
/opt/homebrew/etc
/opt/homebrew/include
/opt/homebrew/lib
/opt/homebrew/sbin
/opt/homebrew/share
/opt/homebrew/var
/opt/homebrew/opt
/opt/homebrew/share/zsh
/opt/homebrew/share/zsh/site-functions
/opt/homebrew/var/homebrew
/opt/homebrew/var/homebrew/linked
/opt/homebrew/Cellar
/opt/homebrew/Caskroom
/opt/homebrew/Frameworks
==> The Xcode Command Line Tools will be installed.
```

Xcode Command Line Tools is mandatory when installing brew tools ? brew tool has dependency on Xcode Command Line Tools ?

**Comment by Bo98** [comment]
On Apple Silicon (Apple M1 etc) you'll need either that or the full Xcode.app. On Intel, things can work without installing anything else.

Though even if you opt for the full Xcode.app, some individual formulae won't install without the Command Line Tools installed (unlike the full Xcode.app, the Command Line Tools install into a fixed, known location) so it is highly recommmended you install it.
